Documentation/dl_build.md

```diff
@@ -64,4 +67,3 @@ If OK is printed, then etcd is working!
 [go]: https://golang.org/doc/install
 [build-script]: ../build
 [cmd-directory]: ../cmd
-
```

Comment:
yeah, but would like to avoid inlined links; also since it's an internal link it should be `op-guide/hardware.md#example-hardware-configurations` instead of the full url since that gunks up the md processing for the site